mava/systems/jax/mappo/system.py

```diff
@@ -86,5 +90,7 @@ def design(self) -> Tuple[DesignSpec, Any]:
             **trainer_process,
             distributor=building.Distributor,
             logger=building.Logger,
+            component_dependency_guardrails=ComponentDependencyGuardrails,
+            component_dependency_debugger=ComponentDependencyDebugger,
```

Comment:
I'm not sure whether this should be the default. What are your thoughts?

Reply:
I think that the debugger should not be there by default, but that the guardrails should. I have just removed the debugger.

Reply:
Sounds good. Thank you